community/modules/compute/gke-node-pool/main.tf

```diff
@@ -109,6 +109,11 @@ resource "google_container_node_pool" "node_pool" {
     }
   }
 
+  timeouts {
+    create = var.timeout_create
```

Comment:
for customer case, since there are 96 nodes being created per nodepool, we have seen GKE taking longer than default value to create and update the nodepools.

Reply:
To add more context, I think it is relevant in combination with #1424 where node creation can now be part of node pool creation (as opposed to nodes being autoscalled after node pool-creation).

Reply:
I added some context to the description.